lisp/ess-r-mode.el

```diff
@@ -1265,44 +1265,58 @@ we flush the cache.")
   "Load/INSTALL/Update ESSR."
   (let* ((pkg-dir (expand-file-name "ESSR" ess-etc-directory))
          (src-dir (expand-file-name "R" pkg-dir)))
-    (if (not (or (and (boundp 'ess-remote) ess-remote)
-                 (file-remote-p (ess-get-process-variable 'default-directory))))
-        (inferior-ess-r-load-ESSR--local pkg-dir src-dir)
-      (inferior-ess-r-load-ESSR--remote pkg-dir src-dir))))
+    (if (or (bound-and-true-p ess-remote)
+            (file-remote-p (ess-get-process-variable 'default-directory)))
+        (inferior-ess-r-load-ESSR--remote pkg-dir src-dir)
+      (inferior-ess-r-load-ESSR--local pkg-dir src-dir))))
 
 (defun inferior-ess-r-load-ESSR--local (pkg-dir src-dir)
+  "Load ESSR.
+PKG-DIR is unused, SRC-DIR should be the location of the R
+directory inside ESSR."
   (let ((cmd (format "local({
                           source('%s/.load.R', local=TRUE) #define load.ESSR
                           load.ESSR('%s')
                       })\n"
                      src-dir src-dir)))
-    (ess-write-to-dribble-buffer (format "load-ESSR cmd:\n%s\n" cmd))
```

Comment:
Why did you remove this? Errors during load will not be propagated to the user.

Reply:
That was a mistake; I'll add it back in.